_posts/2018-04-19-leeds-hack-2018.md

```diff
@@ -0,0 +1,26 @@
+---
+layout:     post
+title:      Leeds Hack 2018 Challenge
+date:       2018-04-19 12:00 +0100
+summary:    Details of our Leeds Hack 2018 challenge. 
+category:   
```

Comment:
This needs to be a valid category, "Community" would probably work for this post or you can create one with `rake createCategory[name]`